src/app/utilities/error-handler.ts

```diff
@@ -1,21 +1,35 @@
-import swal from 'sweetalert2';
+import swal, { SweetAlertOptions } from 'sweetalert2';
 import { SpinnerUtil } from './spinner-utilities';
 import { catchError } from 'rxjs/operators';
 import { Observable, throwError } from 'rxjs';
 
+// TODO: refactor this method to use popupService
 export function displayBackendError(resp: any) {
     const error = resp.error;
 
+    const defaultSettings: SweetAlertOptions = {
+        titleText: 'Warning!',
+        type: 'warning',
+        customClass: 'popup-warning',
+        position: 'top'
+    };
+
     if (error === null) {
         return;
     }
 
     if (error.description !== undefined) {
-        swal('', error.description, 'warning');
+        swal(Object.assign(defaultSettings, {text: error.description}));
     } else if (error.message !== undefined) {
-        swal('', error.message, 'warning');
+        swal(Object.assign(defaultSettings, {text: error.message}));
```

Comment:
Just a notice, consider using [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) instead of `Object.assign` as it's a more modern way of doing it. 
And be aware! Do this only when you know that you are spreading object in a single level. For multi-level spreading and merging, use lodash `merge` function. Read about it [here](https://medium.com/better-programming/how-to-merge-deeply-nested-objects-in-javascript-27e12107480e).

For now, let it stay like this.